Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -276,19 +276,23 @@ public class HTTPClient {
         public var timeout: Timeout
         /// Upstream proxy, defaults to no proxy.
         public var proxy: Proxy?
+        /// Enables ignore NIOSSLError.uncleanShutdown when state is .head or .body, defaults to `false`.
+        public var ignoreNIOSSLUncleanShutdownError: Bool
 
-        public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: Bool = false, timeout: Timeout = Timeout(), proxy: Proxy? = nil) {
+        public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: Bool = false, timeout: Timeout = Timeout(), proxy: Proxy? = nil, ignoreNIOSSLUncleanShutdownError: Bool = false) {
```

Comment:
This change needs to be done differently, I'm afraid. This is a breaking API change, as you are allowed to refer to functions in Swift by their complete set of labels, and this changes the set of labels here. You need to do this:

```swift
public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: Bool = false, timeout: Timeout = Timeout(), proxy: Proxy? = nil) {
    self.init(tlsConfiguration: tlsConfiguration, followRedirects: followRedirects, timeout: timeout, proxy: proxy, ignoreNIOSSLUncleanShutdownError: true)
}

public init(tlsConfiguration: TLSConfiguration? = nil, followRedirects: Bool = false, timeout: Timeout = Timeout(), proxy: Proxy? = nil, ignoreNIOSSLUncleanShutdownError: Bool) {
    self.tlsConfiguration = tlsConfiguration
    self.followRedirects = followRedirects
    self.timeout = timeout
    self.proxy = proxy
    self.ignoreNIOSSLUncleanShutdownError = ignoreNIOSSLUncleanShutdownError
}
```

Reply:
It was reverted.

Reply:
@Lukasa we are not tagged `1.0.0` yet, do you think we can update the signature? It's unlikely imho that someone will be referencing those particular functions, wdyt?

Reply:
Sure, if we're not maintaining a stable API we can do that.